docs/manage/_index.md

```diff
@@ -1,24 +1,41 @@
 ---
-title: "Get started with the Viam app"
+title: "Manage Robots with the Viam app"
 linkTitle: "Manage"
 weight: 22
 type: docs
 no_list: true
-description: "A guide to getting started with app.viam.com."
+description: "A guide to getting started with app.viam.com, a web user interface for building and managing robots with Viam."
 aliases:
   - "/viam/app.viam.com/"
 ---
 
-The [Viam app](https://app.viam.com/) has several key features:
+The [Viam app](https://app.viam.com/) is a web user interface for managing and building robots.
+
+## Create Account and Log In
+
+To get started on the Viam app, you must log in as an authorized user.
+Navigate to [the main page](https://app.viam.com/).
+If you haven't already, toggle to **Sign Up** to create a profile.
+Select **Log In** to log in to the app with your Gmail credentials or your user address and chosen password.
+
+If you forget your password to the app, click **Forgot password** and enter your email address to obtain instructions to reset your password.
+
+## Manage
+
+You do not have to use the app to use Viam's platform to build your robot, but it has several key features to help you get started managing your robots, including configuring robots with more complicated architecture like sub-parts or remotes:
 
 {{< cards >}}
-  {{% card link="/manage/fleet" size="small" %}}
   {{% card link="/manage/configuration" size="small" %}}
+  {{% card link="/manage/fleet" size="small" %}}
   {{% card link="/manage/data" size="small" %}}
   {{% card link="/manage/ml" size="small" %}}
   {{% card link="/manage/parts-and-remotes" size="small" %}}
 {{< /cards >}}
 
 <br>
 
-You can also manage and control your robots from the command line with our [CLI](cli/).
+As an alternative to the app, you can manage and control your robots from the command line with our CLI:
```

Comment:
I think it's only an alternative to managing with the app not to the app

Reply:
@npentrel Ah true, I can see how that would be confusing. Updated